astroid/interpreter/_import/spec.py

```diff
@@ -142,6 +142,31 @@ def find_module(
                 type=ModuleType.C_BUILTIN,
             )
 
+        if submodule_path is not None:
+            search_paths = list(submodule_path)
+        else:
+            search_paths = sys.path
+
+        # First try to find the module using normal search paths. If that fails, fall back to
+        # the frozen module finder.
+        suffixes = (".py", ".pyi", importlib.machinery.BYTECODE_SUFFIXES[0])
+        for entry in search_paths:
+            package_directory = os.path.join(entry, modname)
+            for suffix in suffixes:
+                package_file_name = "__init__" + suffix
+                file_path = os.path.join(package_directory, package_file_name)
+                if os.path.isfile(file_path):
+                    return ModuleSpec(
+                        name=modname,
+                        location=package_directory,
+                        type=ModuleType.PKG_DIRECTORY,
+                    )
+            for suffix, type_ in ImportlibFinder._SUFFIXES:
+                file_name = modname + suffix
+                file_path = os.path.join(entry, file_name)
+                if os.path.isfile(file_path):
+                    return ModuleSpec(name=modname, location=file_path, type=type_)
+
         # sys.stdlib_module_names was added in Python 3.10
         if PY310_PLUS:
             # If the module is a stdlib module, check whether this is a frozen module. Note that
```

Comment:
```suggestion
            # If the module name might be a stdlib module, check whether this is a frozen module. Note that
```

Reply:
I can't comment on the line directly below, but I'd change `imports the module` -> `imports parent modules`